docs/project/governance.md

```diff
@@ -0,0 +1,97 @@
+# pyodide governance and decision-making
+
+The purpose of this document is to formalize the governance process used by the
+pyodide project, to clarify how decisions are made and how the various
+elements of our community interact.
+This document establishes a decision-making structure that takes into account
+feedback from all members of the community and strives to find consensus, while
+avoiding any deadlocks.
+
+Anyone with an
+interest in the project can join the community, contribute to the project
+design and participate in the decision making process. This document describes
+how that participation takes place and how to set about earning merit within
+the project community.
+
+## Roles And Responsibilities
+
+### Contributors
+
+Contributors are community members who contribute in concrete ways to the
+project. Anyone can become a contributor, and contributions can take many forms
+– not only code – as detailed in the {ref}`how_to_contibute`.
+
+### Community members team
+
+The community members team is composed of community members who have permission on
+Github to label and close issues. Their work is
+crucial to improve the communication in the project.
+
+Similarly to what has been decided in the [CPython project](
+https://devguide.python.org/triaging/#becoming-a-member-of-the-python-triage-team),
+any contributor may become a member of this team, after
+showing some continuity in participating in pyodide
+development (with pull requests and reviews).
+Any core developer is welcome to propose a pyodide contributor to join the
+community members team. Other core developers are then consulted: while it is expected
+that most acceptances will be unanimous, a two-thirds majority is enough.
+
+### Core developers
+
+Core developers are community members who have shown that they are dedicated to
+the continued development of the project through ongoing engagement with the
+community. They have shown they can be trusted to maintain pyodide with
+care. Being a core developer allows contributors to more easily carry on
+with their project related activities by giving them direct access to the
+project’s repository and is represented as being a member of the core team on the
+pyodide [GitHub organization](https://github.com/orgs/pyodide/teams/core/members).
+Core developers are expected to review code
+contributions, can merge approved pull requests, can cast votes for and against
+merging a pull-request, and can be involved in deciding major changes to the
+API.
+
+New core developers can be nominated by any existing core developers. Once they
+have been nominated, there will be a vote by the current core developers.
+Voting on new core developers is one of the few activities that takes place on
+the project's private communication channels. While it is expected that most votes
```

Comment:
> private communication channels 

Is there a plan to set up private communication channels? I assume we don't currently have these.

Reply:
Yes, we need to decide which ones. One possibility is ask for creation of a pyodide@python.org mailing list. The other is to use the comment system specific to Github teams, but that would tend to get lost in all the other GH notifications I think.